autogen/main/cluster.tf.tmpl

```diff
@@ -840,6 +840,14 @@ resource "google_container_node_pool" "windows_pools" {
     disk_size_gb    = lookup(each.value, "disk_size_gb", 100)
     disk_type       = lookup(each.value, "disk_type", "pd-standard")
 
+    {% if not beta_cluster %}
```

Comment:
I'm not sure that `ephemeral_storage_local_ssd_config ` will work with google-beta provider. From the documentation I understood that the beta provider has another option: [`ephemeral_storage_config`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/container_cluster#ephemeral_storage_config)

Reply:
Thanks @alvicsam - That nomenclature indicates that `ephemeral_storage_config` is only present in the beta provider.  When not marked as `beta` (such as `ephemeral_storage_local_ssd_config`) the parameter should be in both providers.

Reply:
Regardless it would be good to add the parameter to a (`beta`) example so that it's exercised in the CI tests: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/examples

Reply:
> When not marked as beta (such as ephemeral_storage_local_ssd_config) the parameter should be in both providers.

Yes, you're right. Both `ephemeral_storage_local_ssd_config` and `ephemeral_storage_config` have identical description which is confusing. But [this PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/7483) makes the situation clear (tl;dr: `ephemeral_storage_config` was introduced in google-beta provider and then `ephemeral_storage_local_ssd_config` was added for both providers). I'll make the adjustments and take a look into tests.

Reply:
Great - For testing I would just add the new parameter to an appropriate exiting example with a non-default value.  No need to create a new example for this change.